stack/templates/_helpers.tpl

```diff
@@ -125,3 +125,52 @@ Container probes cannot have both httpGet and tcpSocket fields, so we use omit t
 {{- end }}
 {{- end }}
 
+{{- define "baseDomain" -}}
+{{- $splits := (splitList "." .Values.global.ingress.host) }}
+{{- $last := $splits | last }}
+{{- $secondLast := $splits | initial | last }}
+{{- printf "%s.%s" $secondLast $last -}}
+{{- end -}}
+
+{{- define "oidcProxy.image" -}}
+{{- if typeIs "string" .Values.global.oidcProxy.image }}
+image: {{ .Values.global.oidcProxy.image }}
+{{ else }}
+image: {{ .Values.global.oidcProxy.image.repository }}:{{ .Values.global.oidcProxy.image.tag }}
+{{- end }}
+{{- end }}
```

Comment:
I think `define "initContainer.image"` should be refactored and reused instead of defining this